zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -30,9 +31,37 @@
 class GloriousFoldingAmplifier(DAGNoiseAmplifier):
     """Interface for folding amplifier strategies."""
 
+    def __init__(self, barriers: bool = True, tolerance: float = 0.05) -> None:
+        self._set_barriers(barriers)
+        self._set_tolerance(tolerance)
+
+    ################################################################################
+    ## PROPERTIES
+    ################################################################################
+    @property
+    def barriers(self) -> bool:
+        """Barriers setter"""
+        return self._barriers
+
+    def _set_barriers(self, barriers: bool) -> None:
+        """Set barriers property"""
+        self._barriers = bool(barriers)
+
+    @property
+    def tolerance(self) -> float:
+        """Tolerance setter"""
+        return self._tolerance
+
+    def _set_tolerance(self, tolerance: float) -> None:
+        """Set Tolerance property"""
```

Comment:
This is the real tolerance `setter`
```suggestion
        """Set tolerance property"""
```